go.mod

```diff
@@ -1,5 +1,7 @@
 module sigs.k8s.io/kind
 
+go 1.13
```

Comment:
note: it automatically added something like this. this statement is just a suggestion to go for errors when the build fails, it is NOT a requirement or anything.